sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -251,6 +251,8 @@ test-proxy push -a <path-to-assets.json>
 
 On completion of the push, a newly created tag will be stamped into the `assets.json` file. This new tag must be committed and pushed to your package directory along with any other changes.
 
+☑️ Permission is required for updating the assets repository. If you failed when executing `test-proxy push` command, please [join a partner write team](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/785/Externalizing-Recordings-(Asset-Sync)?anchor=permissions-to-%60azure/azure-sdk-assets%60).
```

Comment:
This is great for internal contributors with access to the linked documentation, however, that is not a public link and non-Microsoft contributors cannot access it.   

We really need an approach for this that allows community contributors to update test assets, but I don't believe that one exists at this time.  Assuming that hasn't changed, can we please add a note that this documentation is "Microsoft Internal" and community contributors will need to work with the reviewers for their PR to get assets merged in?

@JoshLove-msft, @scbedd:  Do we have any update here for community contributor support?

Reply:
Thanks @jsquire I add the notes now. And we may update later if there is a wiki for community contributor. thanks